homeassistant/components/bosch_alarm/sensor.py

```diff
@@ -15,25 +16,57 @@
 from . import BoschAlarmConfigEntry
 from .entity import BoschAlarmAreaEntity
 
+PRIORITY_TYPES = {
```

Comment:
What about "alarm"?

Reply:
Sorry, I meant what about the priority type "alarm", like BURGLARY_ALARM.

Reply:
oh right, i've done alarms as seperate binary sensors

Reply:
and thats in the binary sensor PR

Reply:
Ah, but is there a reason to separate it out?
I think adding an "alarm" state here is the most logical and consistent representation.
In fact, I would argue for adding that state even if we decide to keep the separate binary sensor (I wouldn't).

Reply:
Sure, we can keep the binary sensor, but we should still include the "alert" as one of the states here.
It doesn't make sense for the alarm sensor to be showing "no_issues" when the alarm is going off.

Reply:
have done this now